### PR TITLE
Instructions for online workshop in country field

### PR DESCRIPTION
**Pull request by brownsarahm**
We discussed this for an upcoming workshop, but also should get confirmation that this will not break data in AMY. @klbarnes20 can you help get the right people to review that this is the way to go?

### CODE REVIEW

**Comment by klbarnes20** [comment]
@maneesha @fmichonneau the question was what to put in the country field of an Instructor Training workshop website if it is online. Sarah proposed some additional text. Can you review this and make sure it is the correct way to handle online Instrctor Trainings? Thanks!

**Comment by maneesha** [comment]
I think we are using `W3` if it is online. @fmichonneau  can you confirm?

**Comment by fmichonneau** [comment]
Having separate repositories for workshops and instructor trainings makes it difficult to keep the documentation in sync.

Instructions for workshop website set up are on the [customization page](https://carpentries.github.io/workshop-template/customization/index.html#for-online-workshops).

For workshops, even if the workshop is online, we use the country of the hosting institution so that our statistics (and the maps we generate) on workshop location remain relatively accurate.

Given that the nature of instructor training events is slightly different, and that many events are organized by us and designed for accommodating our global audience, I'm not sure how we want to capture this for our assessment/reporting purposes. If we do want to capture that the events are not tied to a specific geographical location then using "W3" would work as it is how it's represented in AMY.

File: index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: workshop      # DON'T CHANGE THIS.
 root: .               # DON'T CHANGE THIS EITHER.  (THANK YOU.)
-country: "FIXME"      # lowercase two-letter ISO country code such as "fr" (see https://en.wikipedia.org/wiki/ISO_3166-1)
+country: "FIXME"      # "online" for online or lowercase two-letter ISO country code such as "fr" (see https://en.wikipedia.org/wiki/ISO_3166-1)
```
Comment (brownsarahm) [inline index.md]:
```suggestion
country: "FIXME"      # "W3" for centrally organized online trainings or lowercase two-letter ISO country code such as "fr" of the host institution if applicable (see https://en.wikipedia.org/wiki/ISO_3166-1)
```

**Comment by brownsarahm** [comment]
time zone will be in the slug and for IT maybe that's the most relevant, so using w3 makes sense most of the time?

However, making the time zone more prominent in the template is probably a good thing?

**Comment by klbarnes20** [comment]
From a data/assessment standpoint, I think having online events tagged as such is ideal, so it seems like this means using W3 is our best bet as mentioned above.

**Comment by brownsarahm** [comment]
I've changed this to propse using w3 or host country if applicable.  

I'm going to create a separate issue about pulling more updates from the workshop website template over.  we have to do a human-led merge I think because we need so many specific to IT things in this template.

**Comment by brownsarahm** [comment]
Can @klbarnes20 or @maneesha approve this to indicate that it will support clean data in AMY?

**Comment by klbarnes20** [comment]
As @maneesha is more knowledgeable about our data, I'd like her to be the one to approve.

**Comment by maneesha** [comment]
Yes, we can use `W3`

**Comment by brownsarahm** [comment]
thanks!

**Pull request merged**